src/main/java/io/confluent/connect/jdbc/JdbcSourceConnector.java

```diff
@@ -115,25 +115,30 @@ public void start(Map<String, String> properties) throws ConnectException {
   @Override
   public List<Map<String, String>> taskConfigs(int maxTasks) {
     String query = config.getString(JdbcSourceConnectorConfig.QUERY_CONFIG);
+    List<Map<String, String>> taskConfigs;
     if (!query.isEmpty()) {
-      List<Map<String, String>> taskConfigs = new ArrayList<>(1);
+      taskConfigs = new ArrayList<>(1);
```

Comment:
Nit: you can replace this line and the `taskConfigs.add(taskProps)` line with a single call to the `Collections.singletonList(...)`